src/pybind/mgr/telemetry/module.py

```diff
@@ -810,7 +810,7 @@ def gather_perf_counters(self, mode: str = 'separated') -> Dict[str, dict]:
         #           "value": 88814109
         #       },
         #   },
-        all_perf_counters = self.get_all_perf_counters()
+        all_perf_counters = self.get_unlabeled_perf_counters()
```

Comment:
I'd suggest dropping `all_` prefix from this derived `all_perf_counters_by_daemon` variables.